src/client/containers/EditBoardPage/EditBoardPage.container.js

```diff
@@ -1,5 +1,105 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { useUser } from '../../firebase/UserContext';
+import './EditBoardPage.styles.css';
+import ArrowButton from '../../components/ArrowButton/ArrowButton.component';
+import PageTitle from '../../components/PageTitle/PageTitle.component';
+import Input from '../../components/InputComponent/InputComponent';
+import Dropzone from '../DropZone/Dropzone.container';
+import GenericButton from '../../components/GenericButton/GenericButton.component';
+import { ApiError } from '../../ErrorBoundary';
 
-export const EditedBoard = () => (
-  <div className="EditedBoard ">The Edited Board!</div>
-);
+const EditedBoard = () => {
+  const [name, setName] = useState('');
+  const [date, setDate] = useState('');
```

Comment:
When I edit an entry, I expect the starting values to be the current board values. This could be passed in as props or you make a request to fetch the board information.

Reply:
Hi @Midnighter 
from specific board-page they are passing boardinfo as props,
but in the profile page not passing the board info to editboard 
so which is the better way to implement this , write the fetch request or props?

Reply:
If both the profile and the board page have the full information available, I think it's best to pass them as props (avoiding unnecessary requests to the backend is nice).

Reply:
See @senner007 's comment, since this is a new page (a different route) activated via a `Link` we cannot use props. You could pass an object with state in the `Link` as shown [here](https://reactrouter.com/web/api/Link/to-object).

Reply:
That would also allow you to pass information whether the edit page was accessed from the profile or from the board page such that you can return there.